### PR TITLE
Exclude self content share stream index

### PR DESCRIPTION
**Pull request by ltrung**
**Description of changes:**
-  Exclude self content share video stream index.
-  Manually raise videoTileDidUpdate in startContentShare to be backward-compatible with the previous behavior.

**Testing**

1. Have you successfully run `npm run build:release` locally? Yes
2. How did you test these changes? Test with local demo app and verify that content share video is still shown to sharer but addVideoTile is not called from createPeerConnectionTask.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/contentsharecontroller/DefaultContentShareController.ts [hunk 1]
```diff
@@ -106,6 +110,16 @@ export default class DefaultContentShareController
   }
 
   audioVideoDidStart(): void {
+    this.contentShareTile = this.attendeeAudioVideo.videoTileController.addVideoTile();
+    this.contentShareTile.bindVideoStream(
+      this.contentAudioVideo.configuration.credentials.attendeeId,
+      false,
+      this.mediaStreamBroker.mediaStream,
+      null,
+      null,
```
Comment (aburnson) [inline src/contentsharecontroller/DefaultContentShareController.ts:119]: Investigate whether width and height can be derived from MediaStream like it is done here: https://github.com/aws/amazon-chime-sdk-js/blob/0e03fdce362efea84da16af9b28735a72661654a/src/task/CreatePeerConnectionTask.ts#L216

Comment (ltrung) [reply src/contentsharecontroller/DefaultContentShareController.ts]: Added dimension.

File: src/contentsharecontroller/DefaultContentShareController.ts [hunk 2]
```diff
@@ -105,18 +111,66 @@ export default class DefaultContentShareController
...
+        const isContentAttendee = new DefaultModality(attendeeId).hasModality(
+          DefaultModality.MODALITY_CONTENT
+        );
+        const isSelfAttendee =
+          new DefaultModality(attendeeId).base() ===
+          this.attendeeAudioVideo.configuration.credentials.attendeeId;
+        if (isContentAttendee && isSelfAttendee) {
+          if (present && !this.contentShareTile) {
+            const stream = this.mediaStreamBroker.mediaStream;
+            if (stream.getVideoTracks().length > 0) {
```
Comment (aburnson) [inline src/contentsharecontroller/DefaultContentShareController.ts:145]: For readability, to reduce the level of nesting here, you could return early if the conditionals are false.

File: src/contentsharecontroller/DefaultContentShareController.ts [hunk 3]
```diff
@@ -105,18 +111,66 @@ export default class DefaultContentShareController
...
     });
   }
+
+  private setupContentShareEvents(): void {
+    /*
+     *  We use realtimeSubscribeToAttendeeIdPresence instead of audioVideoDidStart because audioVideoDidStart fires
+     *  before the capacity check in Tincan while when realtimeSubscribeToAttendeeIdPresence fires, we know the
+     *  content attendee has been able to pass the capacity check and join the call so we can start the local
+     *  content share video
+     */
```
Comment (aburnson) [inline src/contentsharecontroller/DefaultContentShareController.ts:133]: Prefer single-line `//` over doc-gen style `/* */` for inline comments.

Comment (ltrung) [reply src/contentsharecontroller/DefaultContentShareController.ts]: Good to know. Fixed.

File: src/contentsharecontroller/DefaultContentShareController.ts (hunk 2 above)
Comment (ltrung) [reply src/contentsharecontroller/DefaultContentShareController.ts]: Return early for the two outer loops.

**Pull request merged**